pkg/agent/evictionmanager/plugin/resource/reclaimed_resources.go

```diff
@@ -61,8 +61,11 @@ func NewReclaimedResourcesEvictionPlugin(_ *client.GenericClientSet, _ events.Ev
 		}
 	}
 
-	gracePeriodGetter := func() int64 {
-		return conf.GetDynamicConfiguration().ReclaimedResourcesEvictionConfiguration.GracePeriod
+	deletionGracePeriodGetter := func() int64 {
+		return conf.GetDynamicConfiguration().ReclaimedResourcesEvictionConfiguration.DeletionGracePeriod
+	}
+	thresholdMetToleranceDuration := func() int64 {
```

Comment:
it's better to keep this variable's naming style along with others, maybe "thresholdMetToleranceDurationGetter"?

Reply:
modified, thanks